server/src/main/resources/application.properties

```diff
@@ -2,4 +2,16 @@ spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 server.servlet.encoding.charset=UTF-8
 server.servlet.encoding.force-response=true
 
-atum.web.api.config.some-key=ALFA
+
+postgres = {
```

Comment:
Spring can't properly parse this configuration syntax. I would suggest following approach as in the lines above.

```
postgres.connectionPool="HikariCP"
postgres.dataSourceClass="org.postgresql.ds.PGSimpleDataSource"
postgres.properties.serverName="localhost"
postgres.properties.portNumber="5432"
postgres.properties.databaseName="atum_db"
postgres.properties.user="atum_user"
postgres.properties.password="changeme"
postgres.numThreads=10

Reply:
Can we please confirm this with David?